### ISSUE TITLE
→ Accessibility: Components | Cards

### ISSUE THREAD

**Issue by yceballost** [Telefonica/mistica-design#1493]
#557 

- Incorrect use of `<h1>` as title
- Order of elements (image in media card)
- Order of pretitle title (Use a header to group pretitle, title and subtitle when at least two exist)
- The role of the card should be configurable when is fully clickable 
- Role attribute 
- The title can be a heading or any other tag

https://jira.tid.es/browse/OBVIVO-2635 // Related to https://github.com/Telefonica/mistica-design/discussions/2149
https://jira.tid.es/browse/OBVIVO-2633 // Solved using pretitleAs
https://jira.tid.es/browse/OBVIVO-2638 // Solved in [WEB-1938](https://jira.tid.es/browse/WEB-1938)

## Tasks

- [x] [Update specs](https://www.figma.com/design/tKdPOfcUALzVIh5oizFbm7/branch/pArEzcgxyEMvdvg5YhCpvg/%F0%9F%94%B8-Cards-Specs?m=auto&node-id=9507-6893&t=lxLXzJaj2vB7y98t-1)
- [ ] Design documentation
- [ ] Android ticket
- [ ] iOS ticket

**Comment by aweell** [comment]
Why cards behave like a single element with buttons is explained in the commnets of this PR: https://github.com/Telefonica/mistica-ios/pull/86

**Comment by aweell** [comment]
Web ticket removed since all the items covered by the specs were already adressed in the componente implementation
